packages/web-component/src/app/index.html

```diff
@@ -28,9 +28,18 @@
         flow-id="<flow-id>"
         base-url="<base-url>"
         debug="true"
+        error-transformer="translateError"
       ></descope-wc>
     </div>
     <script>
+      // Translate or modify the error as needed
+      function translateError(errorText) {
```

Comment:
Don't we want to pass the error object we get from the server?
if it contains the error id it might be easier to map